src/solve.jl

```diff
@@ -77,11 +85,12 @@ function DiffEqBase.__init(prob::DiffEqBase.AbstractDDEProblem,
   #   current integration step (so the interpolation is fixed while updating the stages)
   # we wrap the user-provided history function such that function calls during the setup
   # of the integrator do not fail
+  _fh = HistoryWrapper(f, h)
```

Comment:
Should we just wrap `f.f`? To support Jacobians etc properly, we would have to add them to the surround ODEFunction, so we could just get rid of them here?

Reply:
yeah, maybe we do need to wrap the whole thing.